hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/BackendResolverTest.java

```diff
@@ -1,1720 +0,0 @@
-package org.hypertrace.traceenricher.enrichment.enrichers.resolver.backend;
```

Comment:
This big test file was holding tests for various resolvers like Http, Grpc, Mongo, JDBC, ClientSpan. Just moved the tests into their own unit tests file